CMakeLists.txt

```diff
@@ -120,6 +120,8 @@ else()
     message(WARNING "x11 not found. Building without x11 graphics support")
 endif()
 
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror")
```

Comment:
I don't think having `-Wall -Werror` as default is a good idea, because in my experience the warning are quite different based on both the compiler version and the system headers, possibly breaking the build on some systems without a strong reason. (In tests this is already enabled.)

Reply:
Only in the tests is actually a great idea, never thought about it that way. You as a developer still see the warnings, but regular users or on other platforms still are still able to compile. :+1:

Reply:
I've reverted the commit.